packages/ra-auth-cognito-language-english/src/index.ts

```diff
@@ -0,0 +1,22 @@
+import type { CognitoTranslationMessages } from 'ra-auth-cognito';
+
+export const raAuthCognitoEnglishMessages: CognitoTranslationMessages = {
+    ra: {
+        auth: {
+            cognito: {
+                mfa: {
+                    totp: {
+                        association_helper_text:
+                            'Enter the verification code generated by your authenticator app to complete the association.',
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** 
‏
```suggestion
                            'Enter the verification code generated by your TOPT authenticator app to complete the association.',
```

Reply:
I  don't think it's better. Imo people don't know what is an TOPT authenticator but know what is an auhtenticator.